perses/tests/utils.py

```diff
@@ -990,6 +993,9 @@ def generate_solvated_hybrid_test_topology(current_mol_name="naphthalene", propo
     #generate topology proposal
     topology_proposal = proposal_engine.propose(solvated_system, solvated_topology)
 
+    if not propose_geometry:
+        return topology_proposal, solvated_positions
```

Comment:
it's preferable for a function to return the same number of items irrespective of the function choices. 
`return topology_proposal, solvated_positions, None` or `return topology_proposal, solvated_positions, []` makes the function more robust to use